### ISSUE TITLE
ramnode links stale

### ISSUE THREAD

**Issue by ikkini** [bladeswords/benchmark#1]
Ran your script to benchmark a VPS, but it seems the RamNode links are stale. Linode one is still working,  as is the softlayer one.

Ps: this is the benchmark for TheHoster Indonesia (Indonesia Indonesia     Asia    TheHoster       USD     15      OpenVZ  1024    50      Unmetered       1000):

``` console
CPU model :  Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz
Number of cores : 1
CPU frequency :  2143.058 MHz
Total amount of ram : 1024 MB
Total amount of swap : 1024 MB
System uptime :   1:11,
System version: Debian GNU/Linux 7 \n \l
System kernel: Linux indoughnesia 2.6.32-042stab093.5 #1 SMP Wed Sep 10 17:39:49 MSK 2014 x86_64 GNU/Linux
I/O speed :  49.5 MB/s
Testing Network IO to RamNode New York City (USA): 
Testing Network IO to RamNode Atlanta (USA): 
Testing Network IO to RamNode Seatle (USA): 
Testing Network IO to RamNode Netherlands (NL): 
Testing Network IO to Linode Tokyo (JP): 341KB/s
Testing Network IO to Softlayer Singapore (SG): 1.54MB/s
```

**Comment by bladeswords** [comment]
Hi ikkini, thanks for reporting the issue!  It appears that ramnode have changed their directory structure, fixed and added an additional test to an AU server.  Fixed in commit 897c9b5fbfaed49e0c2ff1dcd2f4a85b5f0337b0

Please feel free to comment back with your test results and I will add them to the main page.

**Comment by ikkini** [comment]
Cheers!
Works now (or at least your script does ;)

``` console
# bash bench.sh 
CPU model :  Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz
Number of cores : 1
CPU frequency :  2143.058 MHz
Total amount of ram : 1024 MB
Total amount of swap : 1024 MB
System uptime :   20:16,       
System version: Debian GNU/Linux 7 \n \l
System kernel: Linux indoughnesia 2.6.32-042stab093.5 #1 SMP Wed Sep 10 17:39:49 MSK 2014 x86_64 GNU/Linux
I/O speed :  47.7 MB/s
Testing Network IO to RamNode New York City (USA): 93.4KB/s
Testing Network IO to RamNode Atlanta (USA): 250KB/s
Testing Network IO to RamNode Seatle (USA): 288KB/s
Testing Network IO to RamNode Los Angeles (USA): 267KB/s
Testing Network IO to RamNode Netherlands (NL): 296KB/s
Testing Network IO to Linode Tokyo (JP): 339KB/s
Testing Network IO to Softlayer Singapore (SG): 3.90MB/s
Testing Network IO to Internode Australia (AU): 2.28MB/s
```
